pkg/terminal/command.go

```diff
@@ -1063,10 +1068,49 @@ func setBreakpoint(t *Term, ctx callContext, tracepoint bool, argstr string) err
 	return nil
 }
 
+func setExecutionPoint(t *Term, ctx callContext, argstr string) error {
+	args := strings.SplitN(argstr, " ", 2)
+
+	requestedEp := &api.ExecutionPoint{}
+	locspec := ""
+	if len(args) != 1 {
+		return fmt.Errorf("address required")
+	} else {
+		locspec = argstr
+	}
+
+	locs, err := t.client.FindLocation(ctx.Scope, locspec)
+	if err != nil {
+		locspec = argstr
+		var err2 error
+		locs, err2 = t.client.FindLocation(ctx.Scope, locspec)
+		if err2 != nil {
+			return err
+		}
+	}
+	for _, loc := range locs {
+		requestedEp.Addr = loc.PC
+
+		err := t.client.SetExecutionPoint(requestedEp)
+		if err != nil {
+			return err
+		}
+
+		requestedEp.File = loc.File
+		requestedEp.Line = loc.Line
+		fmt.Printf("execution point jumped to %v\n", formatExecutionPointLocation(requestedEp))
+	}
+	return nil
+}
+
 func breakpoint(t *Term, ctx callContext, args string) error {
 	return setBreakpoint(t, ctx, false, args)
 }
 
+func executionPoint(t *Term, ctx callContext, args string) error {
+	return setExecutionPoint(t, ctx, args)
```

Comment:
I think setExecutionPoint should just be inlined here, since it's not called anywhere else.